database/migrations/2018_07_03_091314_create_pollers_table.php

```diff
@@ -3,34 +3,37 @@
 use Illuminate\Database\Migrations\Migration;
 use Illuminate\Database\Schema\Blueprint;
 
-class CreatePollersTable extends Migration {
+class CreatePollersTable extends Migration
+{
 
-	/**
-	 * Run the migrations.
-	 *
-	 * @return void
-	 */
-	public function up()
-	{
-		Schema::create('pollers', function(Blueprint $table)
-		{
-			$table->integer('id', true);
-			$table->string('poller_name');
-			$table->dateTime('last_polled');
-			$table->integer('devices');
-			$table->float('time_taken', 10, 0);
-		});
-	}
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('pollers', function (Blueprint $table) {
+            $table->integer('id');
+            $table->string('poller_name');
+            $table->dateTime('last_polled');
+            $table->integer('devices');
+            $table->float('time_taken', 10, 0);
+        });
 
+        \DB::statement("ALTER TABLE `pollers` ADD UNIQUE `id` (`id`);");
+        \DB::statement("ALTER TABLE `pollers` CHANGE `id` `id` int(11) NOT NULL auto_increment;");
+        \DB::statement("ALTER TABLE `pollers` ADD PRIMARY KEY (`poller_name`);");
```

Comment:
I've fixed all the validation issues as of now except for this file. The Validation gives this error. 
```
ALTER TABLE `pollers` DROP INDEX `id`, INDEX `id` (`id`);
```
When attempting to add this, it breaks the migration with this error
```
SQLSTATE[42000]: Syntax error or access violation: 1064 
```

Any thoughts on why?